### PR TITLE
Implemented cucumber 3

### PR DESCRIPTION
**Pull request by amencarini**
Reverts amencarini/dummy#2

### CODE REVIEW

**Comment by amencarini** [comment]
Gnec

**Comment by amencarinitest** [comment]
AAAAH

File: features/basic.feature [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Feature: Not much
+  Scenario: Basic step
```
Comment (amencarinitest) [inline features/basic.feature:2]: not so sure.

File: features/step_definitions/basic_steps.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Then(/^something should happen$/) do
+  false.should be_true
+end
```
Comment (amencarinitest) [inline features/step_definitions/basic_steps.rb:3]: Really, this is not nice!

**Comment by amencarinitest** [comment]
POUHHG

**Comment by amencarinitest** [comment]
NOPE

**Comment by amencarinitest** [comment]
arehehtehe

**Comment by amencarinitest** [comment]
arrrhhrehrh

**Comment by amencarinitest** [comment]
goih

**Comment by amencarinitest** [comment]
And another!

**Pull request closed without merging**